### PR TITLE
#NX-18 Nixify subxt code generator

### PR DESCRIPTION
**Pull request by obsessed-cake**
## Issue
https://app.clickup.com/t/20465559/NX-18

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [NX-18 Run docker as part of nix build](https://app.clickup.com/t/20465559/NX-18)

**Review by dzmitry-lahoda — APPROVED** [review]

File: code/utils/subxt-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    mkdir --parent $out/etc/containers $HOME/.config/containers
+
+    ln -s ${pkgs.skopeo.src}/default-policy.json $out/etc/containers/policy.json
+    ln -s ${registriesConf} $out/etc/containers/registries.conf
+    ln -s ${containersConf} $HOME/.config/containers/containers.conf
+    source ${tools}
+
+    podman version
+
+    container_id=$(podman run --rm -d -u$(id -u):$(id -g) -p9944:9944 -p9988:9988 docker.io/composablefi/composable-sandbox:latest)
```
Comment (dzmitry-lahoda) [inline code/utils/subxt-client/default.nix:81]: so you are open ports anyway? and access podman from nix build to ports opened?

Comment (dzmitry-lahoda) [reply code/utils/subxt-client/default.nix:81]: why then not run just pure binaries?

Comment (dzmitry-lahoda) [reply code/utils/subxt-client/default.nix:81]: idea is that you run nodes AND generator insited podman so you no need to open ports. if you have to open ports with podman, than can just run nodes without isolation then?

Comment (dzmitry-lahoda) [reply code/utils/subxt-client/default.nix:81]: so you
1. cp into podman node and subxtclient
2. run node
3. run client 
4. cp back output

**Comment by obsessed-cake** [comment]
No longer relevant.

**Pull request closed without merging**